src/PageLayout/PageLayout.tsx

```diff
@@ -107,7 +107,20 @@ const Root: React.FC<React.PropsWithChildren<PageLayoutProps>> = ({
           }}
         >
           {slots.header}
-          <Box sx={{display: 'flex', flex: '1 1 100%', flexWrap: 'wrap', maxWidth: '100%'}}>{rest}</Box>
+          <Box
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            sx={(theme: any) => ({
+              display: 'flex',
+              flex: '1 1 100%',
+              flexWrap: 'wrap',
+              maxWidth: '100%',
+              [`@media screen and (min-width: ${theme.breakpoints[1]})`]: {
+                columnGap: SPACING_MAP[columnGap],
+              },
+            })}
```

Comment:
Looks like the gap between content and pane collapses on narrow screens. We probably need a `rowGap` in here somewhere.

![No gap between content and pane regions](https://github.com/primer/react/assets/4608155/45ff53d5-0ad0-40f9-9e76-ec7ee34d2973)